src/models/generated/Bus.jl

```diff
@@ -0,0 +1,61 @@
+#=
+This file is auto-generated. Do not edit.
```

Comment:
Do we need this in every file?

Reply:
I think we want to do everything possible to prevent people from changing these files.  I should actually change the files to read-only in the generation script.

Reply:
ok... I think that might be too restrictive while are still developing. I need to make some updates to the structs and test them before putting them in the schema. I don't want to re-generate the whole thing every time.